libs/contact/driver/hubspot/src/config/contact-config.interface.ts

```diff
@@ -2,6 +2,9 @@ import { InjectionToken } from '@angular/core';
 
 import { DaffHubspotConfig } from '@daffodil/driver/hubspot';
 
+/**
+ * The injection token that holds the configuration of the DaffContact feature.
```

Comment:
This doc looks incorrect. This configuration token is specific to the Hubspot driver.